api/src/main/java/org/pmiops/workbench/monitoring/StackdriverStatsExporterService.java

```diff
@@ -0,0 +1,56 @@
+package org.pmiops.workbench.monitoring;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.springframework.stereotype.Service;
+
+/**
+ * Simple wrapper service so we can mock out the initialization step (which depends directly on a
+ * public static method)
+ */
+@Service
+public class StackdriverStatsExporterService {
```

Comment:
TOL: Could we give this an interface, so that all ExporterServices implement that interface? None of the methods seem unique to it being Stackdriver.

Reply:
It specifically wraps the `StackdriverStatsExporter` class from OpenCensus and constructs a `StackdriverStatsConfiguration`. If anyone is going to make a parent interface to the exporters, OpenCensus would be the ones to do it. So we could add an interface, but we're as likely as not to have to break it when we do the next one. Still, I'll keep this in mind if I do a SignalFX exporter.